api/api-doc/column_family.json

```diff
@@ -380,9 +380,9 @@
          "operations":[
             {
                "method":"GET",
-               "summary":"check if the auto compaction disabled",
+               "summary":"check if the auto compaction enabled",
```

Comment:
you negate the logic, will the jmx still work correctly?

Reply:
@glommer "check if the auto_compaction property is enabled for a given table" :+1:

Reply:
@amnonh you are right. Can I also patch JMX? Or it is not allowed to break API compatibility? What I am concerning of is that positive API name (/column_family/autocompaction/{name}) expected negated value, while the neutral statement must mean IF IT IS ENABLED.

This must break JMX that did not work anyway: https://github.com/scylladb/scylla-jmx/blob/ff0723abc6c2dd23313f93cd5ec396ffd731d743/src/main/java/org/apache/cassandra/db/ColumnFamilyStore.java#L321
Because this API returned `true`. 

I will check node tool, because early checking does not make it clear.  It uses 2 isAutoCompactionDisabled impl, one from MBean and one something different: https://github.com/scylladb/scylla-tools-java/blob/6126d38a1fcb6c2032a6ccdd601b141532c3ffa7/src/java/org/apache/cassandra/db/ColumnFamilyStore.java#L2345 . I will take a look.

But as fas as I understand the API returned invalid value anyway, we can break back compact. ?